WTWSMS/decisions/WtWSMS_realm_decisions.txt

```diff
@@ -186,7 +202,13 @@ decisions = {
 		only_playable = yes
 		is_high_prio = yes
 		potential = {
-			religion_group = christian 
+			OR = {
+				religion_group = christian
+				religion_group = hellenic_pagan
+				religion_group = neoplatonist
+				has_romance_culture_trigger = yes
+				culture = roman
+			}
```

Comment:
It is already done in that way, except for leaving out non-Christians, which is intended, I believe.